src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,304 @@
+
+.container {
+    background-color: #ffffff;
+    display: flex;
+    flex-direction: column;
+    width: 387px;
+    padding: 1rem;
+    height: 1002px;
+    border-radius: 1.2rem;
+}
+.container .title{
+    background-color: #ffffff;
+    width: 320px;
+    height: 53px;
+    margin-top: 1.5rem;
+}
+.container  h2{
+    margin-top: 0.7rem;
+    margin-left: 1.5rem;
+    font-family: 'Times New Roman', Times, serif;
```

Comment:
Remove font family, this applies for the whole CSS document, because it appears very often
```suggestion
```